scripts.js

```diff
@@ -1 +1,16 @@
-console.log('Hello World!');
\ No newline at end of file
+function getGreeting(name, surname) {
+    console.log(`Hi ${name} ${surname}! Good to see you`)
+}
+const greeting = getGreeting('John', 'Smith');
+
+console.log(greeting);
+
+function isThisMyName(name) {
+    if (name === 'Szymon') {
```

Comment:
What type of data is returning this expression? `name === 'Szymon'`? @szymon-czudowski

Reply:
This function can be shorter, the above question may lead you to the better solution